app/src/containers/pages/Home/Home.tsx

```diff
@@ -61,14 +55,6 @@ export const Home = () => {
                 </Button>
               </Link>
             </Col>
-
-            <Col className="gutter-row" span={12}>
```

Comment:
maybe replace this with something FHIR supports to even the count out?

Reply:
Will refactor the landing view in issue https://github.com/opensrp/web/issues/1151.

Note even if we even the count in the code, The ui will still show up any which way depending on the disabled modules.